===================================================================
RCS file: /horde/cvs/passwd/config/backends.php.dist,v
retrieving revision 1.14.2.3
retrieving revision 1.14.2.4
diff -p --unified=3 -r1.14.2.3 -r1.14.2.4
--- config/backends.php.dist	2003/02/05 15:48:13	1.14.2.3
+++ config/backends.php.dist	2004/06/24 08:02:36	1.14.2.4
@@ -147,6 +147,9 @@ $backends['sql'] = array (
     'params' => array(
         'phptype'    => 'mysql',
         'hostspec'   => 'localhost',
+        'protocol'   => 'tcp',
+        // 'port'       => 3306,
+        // 'socket'     => '/tmp/mysql.sock',
         'username'   => 'horde',
         'password'   => '',
         'encryption' => 'md5-hex',
@@ -182,6 +185,9 @@ $backends['vpopmail'] = array (
     'params' => array(
         'phptype'    => 'mysql',
         'hostspec'   => 'localhost',
+        'protocol'   => 'tcp',
+        // 'port'       => 3306,
+        // 'socket'     => '/tmp/mysql.sock',
         'username'   => '',
         'password'   => '',
         'encryption' => 'crypt',
